src/pages/Admin/Admin.tsx

```diff
@@ -0,0 +1,40 @@
+import Layout from "../../components/Layout/Layout";
+import "./Admin.scss";
+import Header from "../../components/Header/Header";
+import Button from "../../components/Button/Button";
+import placeHolderPFP from "/src/assets/images/placeHolderPFP.svg";
+
+const Admin = () => {
+  const handleSignOut = () => {
+    return;
+  };
+
+  return (
+    <Layout>
+      <Header title={"Admin"} />
+      <div className="admin-page">
+        <div className="admin-page__content">
+          <div className="admin-page__content--photo">
+            <img
+              className="admin-page__content--image"
+              src={placeHolderPFP}
+              alt="your photo"
+            />
+          </div>
+        </div>
+        <div className="add-events-button">
+          <Button label={"Add Events"} onClick={handleSignOut} />
+        </div>
+        <div className="manage-events-button">
+          <Button label={"Manage Events"} onClick={handleSignOut} />
```

Comment:
What is managing events?

Reply:
we were planning on creating a page where it would display all events and let the admin delete an event

Reply:
Let's remove the button as it isn't adding any functionality